generate/generate.go

```diff
@@ -644,10 +644,14 @@ func (g *Generator) convertParameter(tuple *types.Tuple) (*string, error) {
 		return nil, fmt.Errorf("multiple parameters are not supported")
 	}
 	param := tuple.At(0).Type()
-	_, _, tname := g.convertType(param)
+	_, label, tname := g.convertType(param)
```

Comment:
hmm, maybe we should have made it harder to skip part of the returned info...

Reply:
Not too sure what we could do to make it harder to skip?

Reply:
Not sure either, hence the "hmm, maybe" :) just thinking outloud.